config/routes.rb

```diff
@@ -7,6 +7,13 @@ def edit_step(name)
 end
 # :nocov:
 
+
+def show_step(name)
```

Comment:
Wrap this method in the above `:nocov:` annotation, like we did with `edit_step`, as we are not testing directly this methods, and otherwise it reduces our test coverage.